Nodejs/Product/Nodejs/Project/NodejsProjectLauncher.cs

```diff
@@ -152,6 +158,18 @@ private void StartAndAttachDebugger(string file, string nodePath)
             dbgInfo.bstrExe = $"\01";
 
             AttachDebugger(dbgInfo);
+
+            if (startBrowser && uri != null)
+            {
+                OnPortOpenedHandler.CreateHandler(
+                    uri.Port,
+                    shortCircuitPredicate: () => process.HasExited,
+                    action: () =>
+                    {
+                        VsShellUtilities.OpenBrowser(url, (uint)__VSOSPFLAGS.OSP_LaunchNewBrowser);
```

Comment:
Does this take any notice of the browser selected in the "Start debugging" drop-down? I noticed it was always launching the default browser, no matter what was selected here (though that may be a different bug to address).